app/javascript/article-form/elements/moreConfig.jsx

```diff
@@ -94,6 +101,21 @@ export default class MoreConfig extends Component {
   }
 }
 
+const noop = () => {};
+
+MoreConfig.defaultProps = {
+  onSaveDraft: noop,
+  onConfigChange: noop,
+};
+
 MoreConfig.propTypes = {
   onExit: PropTypes.func.isRequired,
+  onSaveDraft: PropTypes.func,
+  onConfigChange: PropTypes.func,
+  passedData: PropTypes.shape({
+    series: PropTypes.string,
+    canonicalUrl: PropTypes.string,
+    published: PropTypes.bool,
+    allSeries: PropTypes.array,
```

Comment:
Just FYI: in `orgSettings.jsx` eslint didn't allow me to simply use `PropTypes.array` but if it is part of `PropTypes.shape` (this file), it's fine. :)